### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by neil-bennett-ellevation**
I added a new section with a list of items

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # fun-with-git
 
 Just a demo of how to use git with a flow and other fun things!
+
+## Make some changes
```
Comment (neil-bennett-ellevation) [inline README.md:5]: What do you think @ginniss2022 ?

**Comment by neil-bennett-ellevation** [comment]
Oops I didn't mean to open this yet

File: README.md (hunk 1 above)
Comment (ginniss2022) [reply README.md:5]: Needs more work

Comment (neil-bennett-ellevation) [reply README.md:5]: @ginniss2022 updated it...take a look!

**Review by ginniss2022 — APPROVED** [review]

**Pull request merged**